### DISCUSSION TITLE
Storage size problem

### DISCUSSION THREAD

**Discussion by bgramaje** [feilipu/Arduino_FreeRTOS_Library#66] [Q&A]
**Describe the issue**
The Sketch uses 47,228 bytes (146%) of the program storage space. The maximum is 32256 bytes.
text section exceeds available space in board
Global variables use 1574 bytes (76%) of dynamic memory, leaving 474 bytes for local variables. The maximum is 2048 bytes.
Very cool program: visit http://www.arduino.cc/en/Guide/Troubleshooting#size to see how to reduce it.
Compiling error for the Arduino Uno board.

**Expected behavior**
I would like to be able to compile programs cause I cant compile anything.

**Screenshots**
![image](https://user-images.githubusercontent.com/56760866/78041187-12b04380-7379-11ea-9620-cf57683e8172.png)

**Desktop (please complete the following information):**
 - OS: [Windows 10]

**Additional context**

**Comment by feilipu** [comment]
I have tried to replicate the issue (see screenshot below) using the most recent IDE version (but that shouldn't matter), and I can't.

Perhaps you can try installing the IDE again? But, I can't advise that will actually help.

![Blink_AnalogRead](https://user-images.githubusercontent.com/3955592/78329742-34563880-75ce-11ea-843e-28426161fe2a.png)

**Comment by feilipu** [comment]
Please reopen if issue persists.
